core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -300,7 +300,6 @@ private[spark] object JettyUtils extends Logging {
         connector.setReuseAddress(!Utils.isWindows)
          // spark-45248: set the idle timeout to prevent slow DoS
         connector.setIdleTimeout(8000)
-        connector.setStopTimeout(stopTimeout)
```

Comment:
Jetty 10 does not support set timeout for graceful shutdown?

Reply:
I'm not sure, but, looks like the method is still on `Server` so maybe it is just not separately settable here

Reply:
Yes, I have checked. 
StopTimeout is still valid for `Server` but no longer for `Connector` as it is now transited to `Graceful.LifeCycle` and only certain components have it. 
For a detailed explanation, see below the MR from the jetty:
https://github.com/jetty/jetty.project/pull/4482